src/main.cpp

```diff
@@ -145,13 +145,35 @@ int T_main(int argc, TCHAR* argv[])
 		return -1;
 	}
 
-	PEImage img;
-	if (!img.loadExe(argv[1]))
-		fatal(SARG ": %s", argv[1], img.getLastError());
-	if (img.countCVEntries() == 0 && !img.hasDWARF())
-		fatal(SARG ": no codeview debug entries found", argv[1]);
+	PEImage exe, dbg, *img = NULL;
+	TCHAR dbgname[MAX_PATH];
 
-	CV2PDB cv2pdb(img);
+	if (!exe.loadExe(argv[1]))
+		fatal(SARG ": %s", argv[1], exe.getLastError());
+	if (exe.countCVEntries() || exe.hasDWARF())
+		img = &exe;
+	else
+		if (exe.areDebugSymbolsStripped())
+		{
+			T_strcpy(dbgname, argv[1]);
+			TCHAR *pDot = T_strrchr(dbgname, '.');
+			if (!pDot || pDot <= T_strrchr(dbgname, '/') || pDot <= T_strrchr(dbgname, '\\'))
+				T_strcat(dbgname, TEXT(".dbg"));
+			else
+				T_strcpy(pDot, TEXT(".dbg"));
+
+			if (!dbg.loadExe(dbgname)) 
+				fatal(SARG ": %s", dbgname, dbg.getLastError());

```

Comment:
The most likly cause for failing here is that the DBG file doesn't exist, so the error should still reference the exe file.

Reply:
Another option is a malformed DBG.